Parse/ParseInstallation.Unity.cs

```diff
@@ -43,11 +43,13 @@ public partial class ParseInstallation : ParseObject {
     public int Badge {
       get {
         if (PlatformHooks.IsIOS) {
-				  PlatformHooks.RunOnMainThread(() => {
+#if IOS
```

Comment:
This flag make sure this part of the code only included in **Xamarin iOS**, not all iOS platform. Hence, this will not fix the Unity iOS issue.